ArduPlane/AP_Arming.cpp

```diff
@@ -136,6 +136,30 @@ bool AP_Arming_Plane::ins_checks(bool display_failure)
 
 bool AP_Arming_Plane::arm_checks(AP_Arming::Method method)
 {
+    if (method == AP_Arming::Method::RUDDER) {
+        const AP_Arming::RudderArming arming_rudder = get_rudder_arming_type();
+
+        if (arming_rudder == AP_Arming::RudderArming::IS_DISABLED) {
+            //parameter disallows rudder arming/disabling
+            check_failed(true, "Rudder arming disabled");
```

Comment:
Doesn't  this mean I now get warnings abut rudder arming being disabled when doing something like a RC calibration? If I selected rudder arming disabled, I really don't ever want to see a message print out about it. This seems like a regression.

Reply:
Yes, it does.  OTOH, you shouldn't be holding rudder over for *that* long on any vehicle you don't want to arm IMO.

I'll remove the message and add a comment about why we're not sending a message.

Reply:
This was done.